data/resources/style.css

```diff
@@ -251,27 +253,32 @@ messagemedia overlay > messageindicators, messagesticker messageindicators {
 }
 
 .sender-text-cyan {
-  color: #2996ad;
+    color: #2996ad;
 }
 
 .sender-text-blue {
   color: #168acd;
 }
 
 .sender-text-pink {
-  color: #cd4073;
+    color: #cd4073;
 }
 
 /* Placeholder */
 .message-entry > overlay > label {
   margin: 0 9px;
 }
 
+.message-entry > overlay > entry > image {
+  margin-bottom: 9px;
+}
+
 .message-entry > overlay > entry {
-  padding: 0;
+  padding: 0 9px;
+  border-radius: 12px;
```

Comment:
What do you think about using SCSS Instead of regular CSS?
it has variables and other features

Reply:
> What do you think about using SCSS Instead of regular CSS?

I think it's a bit of an overkill, but if our CSS becomes too complicated it might be worth start using it. Currently I'm kinda happy of the current CSS, especially after the reorganization happened in #381.